src/panels/lovelace/hui-root.ts

```diff
@@ -697,13 +731,19 @@ class HUIRoot extends LitElement {
 
         ha-app-layout {
           min-height: 100%;
+          background: var(--lovelace-background);
         }
         ha-tabs {
-          margin-left: max(env(safe-area-inset-left), 24px);
-          margin-right: max(env(safe-area-inset-right), 24px);
+          width: 100%;
+          height: 100%;
```

Comment:
Should this be
```suggestion
          height: var(--header-height);
```

Reply:
Ah, right. This should have stayed 100% and bram just fixed it. When in edit mode, ha-tabs ends up in div sticky. So it needs to be 100%